src/config/rbd/translation.json

```diff
@@ -19,5 +19,88 @@
+  "1-month Standardized Precipitation Index calculated from dekadal CHIRPS data": "Indice de précipitations standardisé sur 1 mois calculé à partir des données décadaires CHIRPS",
+  "3-month Standardized Precipitation Index calculated from dekadal CHIRPS data": "Indice de précipitations standardisé sur 3 mois calculé à partir des données décadaires CHIRPS",
+  "6-month Standardized Precipitation Index calculated from dekadal CHIRPS data": "Indice de précipitations standardisé sur 6 mois calculé à partir des données décadaires CHIRPS",
+  "9-month Standardized Precipitation Index calculated from dekadal CHIRPS data": "Indice de précipitations standardisé sur 9 mois calculé à partir des données décadaires CHIRPS",
+  "1-year Standardized Precipitation Index calculated from dekadal CHIRPS data": "Indice de précipitations standardisé sur 1 an calculé à partir des données décadaires CHIRPS",
+  "Number of consecutive days with less than 2 mm precipitation in the last 30 days": "Nombre de jours consécutifs avec moins de 2 mm de précipitations au cours des 30 derniers jours",
+  "Longest number of consecutive days with less than 2 mm precipitation in the last 30 days": "Plus grand nombre de jours consécutifs avec moins de 2 mm de précipitations au cours des 30 derniers jours",
+  "Total number of heavy rain days (rainfall > 75th percentile) within last 30 days of dekad": "Nombre total de jours de fortes pluies (pluies > 75e centile) au cours des 30 derniers jours de la décade",
+  "Total number of heavy rain days (rainfall > 90th percentile) within last 30 days of dekad": "Nombre total de jours de fortes pluies (pluies > 90e centile) au cours des 30 derniers jours de la décade",
+  "Total number of heavy rain days (rainfall > 95th percentile) within last 30 days of dekad": "Nombre total de jours de fortes pluies (pluies > 95e centile) au cours des 30 derniers jours de la décade",
+  "Longest consecutive number of heavy rain days (rainfall > 75th percentile) within last 30 days of dekad": "Plus long nombre consécutif de jours de fortes pluies (pluies > 75e centile) au cours des 30 derniers jours de la décade",
+  "Normalized Difference Vegetation Index (NDVI) derived from MODIS TERRA/AQUA": "Indice de végétation par différence normalisée (NDVI) dérivé de MODIS TERRA/AQUA",
+  "Normalized Difference Vegetation Index (NDVI) anomaly when compared to the long-term average": "Anomalie de l'indice de végétation par différence normalisée (NDVI) par rapport à la moyenne à long terme",
+  "Daytime land surface temperature": "Température diurne de la surface terrestre",
+  "Daytime land surface temperature anomaly when compared to long-term average": "Anomalie de la température diurne de la surface terrestre par rapport à la moyenne à long terme"
```

Comment:
`de la surface terrestre` can be simplified -> `au sol`

Reply:
updating, thanks!